bindata/network/ovn-kubernetes/common/002-rbac.yaml

```diff
@@ -133,3 +133,178 @@ subjects:
 - kind: ServiceAccount
   name: ovn-kubernetes-node
   namespace: openshift-ovn-kubernetes
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: openshift-ovn-kubernetes-node-extra
```

Comment:
After talking on slack this is a copy of openshift-ovn-kubernetes-controller ClusterRole defined here: https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/common/003-rbac-controller.yaml#L9-L128.
I don't think we should do that, if we need more permissions for ovnkube-node we should add them to openshift-ovn-kubernetes-node clusterrole

Reply:
True. I'm testing that here: https://github.com/openshift/cluster-network-operator/pull/1905/commits/9761dd8a4e47eb78f350c2b86ce4df1894f69b77

Reply:
Done, I've moved it to this PR as well